docs/userGuide/syntax/thumbnails.mbdf

```diff
@@ -0,0 +1,59 @@
+## Thumbnails
+
+**A `thumbnail` component allows you to insert thumbnails using text, images, or icons.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<thumbnail circle src="images/deer.jpg" size="100"/>
+<thumbnail circle text=":book:" background="#333" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" font-color="#339" border="3px solid #339" size="100"/>
+<thumbnail circle text=":fas-book:" font-color="darkgreen" size="100"/>
+
+<thumbnail src="images/deer.jpg" size="100"/>
+<thumbnail text=":book:" background="#333" size="100"/>
+<thumbnail text="___CS___" background="#ccf" font-color="#339" border="3px solid #339" size="100"/>
+<thumbnail text=":fas-book:" font-color="darkgreen" size="100"/>
+```
+</span>
+<span id="output">
+<thumbnail circle src="images/deer.jpg" size="100"/>
+<thumbnail circle text=":book:" background="#333" size="100"/>
+<thumbnail circle text="___CS___" background="#ccf" font-color="#339" border="3px solid #339" size="100"/>
+<thumbnail circle text=":fas-book:" font-color="darkgreen" size="100"/>
+
+<thumbnail src="images/deer.jpg" size="100"/>
```

Comment:
your `src` is wrong for all the thumbnails.

Since `thumbnails.mbdf` is in `userGuide/syntax/` and your `image` folder is at the same level as `userGuide`. You are supposed to use `../../images/deer.jpg` as your `src`.